hatchet_sdk/hatchet.py

```diff
@@ -110,13 +110,24 @@ def inner(func):
 class Hatchet:
     client: ClientImpl
 
+    @classmethod
+    def from_environment(cls, defaults: ClientConfig = ClientConfig(), **kwargs):
+        return cls(client=new_client(defaults), **kwargs)
+
+    @classmethod
+    def from_config(cls, config: ClientConfig, **kwargs):
+        return cls(client=new_client_raw(config), **kwargs)
+
     def __init__(
         self,
-        debug=False,
+        client: Optional[ClientImpl] = None,
         config: ClientConfig = ClientConfig(),
+        debug=False,
```

Comment:
@colonelpanic8 i think i'd like to keep param order intact here. although all examples use Hatchet(debug=True) folks might be setting Hatchet(true) so this would be a breaking change.